org.gnome.Raider.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.gnome.Raider
+runtime: org.gnome.Platform
+runtime-version: '1.0'
```

Comment:
this runtime version doesn't exist.

Reply:
Still finding my feet, I realized that Meson already provided me with a manifest, I didn't have to spend 2 weeks creating a terrible one...

Reply:
I will change the manifest :)